rampwf/utils/tests/test_testing.py

```diff
@@ -11,15 +11,14 @@
 def test_submission_all_kits():
     tmp_dir = mkdtemp()
     try:
-        test_notebook = True
         for kit in RAMP_KITS_AVAILABLE:
             kit_dir = fetch_ramp_kit(kit, ramp_kits_home=tmp_dir)
             assert_submission(
                 ramp_kit_dir=kit_dir, ramp_data_dir=kit_dir,
-                submission='starting_kit')
-            # testing assert_notebook on first kit
-            if test_notebook:
+                submission='starting_kit', is_pickle=True, save_y_preds=True)
```

Comment:
retrain=True as well?

Reply:
I actually rethought this a bit. For titanic we test `assert_notebook` and the optional switches (to test the script itself). For the others, we don't need the switches (remember the kits' role here is to test functionalities of rampwf; then in the travis of every kit, we should test the different ways of calling the script for that particular kit).